File: lista1sala/1.py
tempo=float(input("digite o tempo gasto na viagem(em horas):"))
vm= float(input( "digite a velocide media durante a viagem (em km/h)"))
distancia= tempo*vm
lu= distancia/12
print("velocidade media", vm, "km/h")
print ("tempo gasto na viagem", tempo, "horas")
print("distancia percorrida", distancia, "km")
print ("quantidade de litros usados na viagem", '%.2f' %lu, "litros")

File: lista1sala/1.c

#include <stdio.h>
int main(){
    float tempo, vm, dis, lu;
    printf("digite o tempo gasto na viagem(em horas)");
    scanf("%f", &tempo);
    printf("digite a velocidde média da viagem(em km/h)");
    scanf("%f", &vm);
    dis=tempo*vm;
    lu=dis/12;
    printf("velocidade média: %.2f km/h\n", vm);
    printf("tempo gasto na viagem: %.2f horas\n", tempo);
    printf("quantidade de litros uados na viagem: %.2f litros\n", lu);

    return 0;
}